src/components/Quota.js

```diff
@@ -0,0 +1,7 @@
+const Quota = () => (
+  <div className="Quota">
+    <h1>Quota</h1>
+  </div>
```

Comment:
- [ ] You could add a quote to your quotes page as shown in the given design.

Your design
![image](https://user-images.githubusercontent.com/81932478/190454296-051c3f0a-a743-4743-9278-1a8b2869d07f.png)

Given template
![image](https://user-images.githubusercontent.com/81932478/190454399-7ee7de68-ce0f-4ed1-88ea-36f70197460d.png)